### PR TITLE
use flux preds for annual/multi-year, conc preds for plots

### PR DESCRIPTION
**Pull request by aappling-usgs**
@wdwatkins - i think this should fix the issue that plotEGRET requires conc preds without breaking anything you've already done (e.g., calculating annual & multi-year fluxes). do you think so, too?

### CODE REVIEW

File: batch.R [hunk 1]
```diff
@@ -158,15 +158,15 @@ for(i in 1:nrow(fileDF)) {
...
+  pconc_interp <- predictSolute(interpRect, "conc", siteQ, se.pred = TRUE, date = TRUE)
+  pconc_comp <- predictSolute(comp, "conc", siteQ, se.pred = TRUE, date = TRUE)
+  pflux_rload <- predictSolute(rloadest5param, "flux", siteQ, se.pred = TRUE, date = TRUE)
+  pflux_interp <- predictSolute(interpRect, "flux", siteQ, se.pred = TRUE, date = TRUE)
+  pflux_comp <- predictSolute(comp, "flux", siteQ, se.pred = TRUE, date = TRUE)
   nPreds <- nrow(siteQ)
-  allPreds <- bind_rows(pred_rload, pred_interp, pred_comp)
-  allPreds$model <- c(rep("rloadest",nPreds), rep("interp",nPreds), rep("composite", nPreds))
+  allPreds <- bind_rows(pconc_rload, pconc_interp, pconc_comp)
+  allPreds$model <- rep(c('rloadest','interp','composite'), each=nPreds)
```
Comment (aappling-usgs) [inline batch.R:170]: I changed this line just in passing. Did you know about the `each` argument to `rep`? I like it for cases like this one.

Comment (wdwatkins) [reply batch.R:170]: No I didn't 👍

**Review by wdwatkins — APPROVED** [review]
Should be good 👍

**Comment by wdwatkins** [comment]
I think whether git detects a change in the output csvs would also test this?

**Comment by aappling-usgs** [comment]
ooh, good point. i didn't commit any of the outputs, but many of them changed. i'll look at that...

**Comment by aappling-usgs** [comment]
```diff
 "site.id","constituent","multi_year_avg.rloadest","SE.rloadest","CI_lower.rloadest","CI_upper.rloadest","multi_year_avg.interpolation","SE.interpolation","CI_lower.interpolation","CI_upper.interpolation","multi_year_avg.composite","SE.composite","CI_lower.composite","CI_upper.composite"
-"MOGU02900","NO3",7204.91776490087,71.3301293376645,7065.11328038646,7344.72224941527,6894.29163267447,65.0250086932982,6766.84495754121,7021.73830780774,6666.04866523874,98.1244072812874,6473.72836096308,6858.3689695144
-"ORIZ02900","NO3",216.776805089324,2.27382511895727,212.320189749025,221.233420429623,196.936377956194,4.20615201090545,188.692471501318,205.180284411069,191.109580016952,3.5513014555574,184.149157065814,198.070002968089
-"RONC02800","NO3",144.800957507442,1.24859956987963,142.353747319366,147.248167695519,187.738090615769,1.0442258445642,185.691445568698,189.784735662841,185.117511527659,1.0334542900084,183.091978339574,187.143044715744
+"MOGU02900","NO3",7204.91776490087,71.3301293376645,7065.11328038646,7344.72224941527,6894.29163267447,65.0250086932982,6766.84495754121,7021.73830780774,6666.04866523874,97.9471658834773,6474.07574771936,6858.02158275813
+"ORIZ02900","NO3",216.776805089324,2.27382511895727,212.320189749025,221.233420429623,196.936377956194,4.20615201090545,188.692471501318,205.180284411069,191.109580016952,3.57820732447422,184.096422531765,198.122737502138
+"RONC02800","NO3",144.800957507442,1.24859956987963,142.353747319366,147.248167695519,187.738090615769,1.0442258445642,185.691445568698,189.784735662841,185.117511527659,1.00733212664663,183.143176838961,187.091846216357
```
those look like very small differences to me. small is good. that they're not 0 is probably due to the random component of the uncertainty estimation for composite models. i'm going to add a `set.seed` and commit some of the output files so that next time the git diffs can be even easier to read. (now that i consider it, i don't know whether the same seed on both our computers will generate the same output. we'll find out.)

**Comment by aappling-usgs** [comment]
oops, shoulda let you merge that. sorry.

**Pull request merged**